LICENSE

```diff
@@ -1,34 +1,25 @@
-Files in this project are works of the US Government 
-and cannot be copyrighted, unless explicitly stated otherwise.
+SPDX license identifier: BSD-3-Clause
+Copyright (c) 2012-2017, Red Hat, Inc.
```

Comment:
Many contributors might have @redhat.com EMails.... but many don't. Will ``copyright Red Hat`` make this feel like to much of a Red Hat-only project? Other OS' do ship SSG:

https://launchpad.net/ubuntu/+source/scap-security-guide
https://launchpad.net/ubuntu/+source/scap-security-guide/0.1.31-5

Not sure how this is handled for other projects (that aren't a foundation, like OpenStack).

Reply:
Usually people add more and more copyright clauses over time as more people contribute.

Reply:
@mpreisler @shawndwells we can always make sure to have other projects add their copyright clauses by pointing it out in documentation and other means.

Or create the opencontrol/opencompliance foundation...

Reply:
If at all possible I think we should avoid any copyright assignment / foundation kind of thing. That's what will keep people from contributing because it's a pain to sign and scan documents.

Reply:
I have seen some `BSD-3` licenses without copyright. So we could remove the copyright *I think*, or just make sure that people are aware that they can add their own.